scripts/setup/mac/install_prereqs

```diff
@@ -75,10 +77,13 @@ tinyxml
 tinyxml2
 vtk@8.0
 yaml-cpp
-zlib
 EOF
 )
 
+if [[ ! -f "${HOME}/.bazelrc" ]] || ! grep -q 'build --python_path=/usr/local/opt/python/libexec/bin/python' "${HOME}/.bazelrc"; then
```

Comment:
Shambhala has no bazel projects so it does not need this. If however, you're just trying to match the scripts exactly to make updates easier till we have a better option, that's ok.

Reply:
No, it does because of the CMake external build. Without this, Shambhala will break the moment the system protobuf PR is merged.

Reply:
Thanks for picking this up btw, was going to find it difficult to catch time for it today.

Reply:
Uses the python for the protobuf generation?

Reply:
Yes, exactly.

Reply:
Drake needs it for build and test, we just need it for build.

Reply:
On the same page now, thanks!